CMakeLists.txt

```diff
@@ -221,7 +224,7 @@ install(TARGETS ${PROJECT_NAME}_shared EXPORT ${PROJECT_NAME}Config
   PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
 )
 
-install(EXPORT ${PROJECT_NAME}Config DESTINATION share/${PROJECT_NAME}/cmake)
+install(EXPORT ${PROJECT_NAME}Config DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME})
```

Comment:
Why not CMAKE_INSTALL_DATADIR as it is 'share' by default?

Reply:
`share/cmake` is for arch-independent CMake projects, while `lib(64)/cmake` is for compiled ones.

Reply:
Ok, fine. Please split -s change to separate PR so all other changes can be approved merged here.

Reply:
done